### ISSUE TITLE
Daily Hacker News 01-05-2026

### ISSUE THREAD

**Issue by blka** [blka/daily-hackernews#388]
# Daily Hacker News

1. [**Grok 4.3**](https://docs.x.ai/developers/models/grok-4.3) `docs.x.ai` [`comments`](https://news.ycombinator.com/item?id=47972447)
2. [**Auto Polo**](https://en.wikipedia.org/wiki/Auto_polo) `en.wikipedia.org` [`comments`](https://news.ycombinator.com/item?id=47936184)
3. [**How Mark Klein told the EFF about Room 641A [book excerpt]**](https://thereader.mitpress.mit.edu/the-whistleblower-who-uncovered-the-nsas-big-brother-machine/) `thereader.mitpress.mit.edu` [`comments`](https://news.ycombinator.com/item?id=47965060)
4. [**New copy of earliest poem in English, written 1,3k years ago, discovered in Rome**](https://www.tcd.ie/news_events/articles/2026/caedmons-hymn-discovery/) `www.tcd.ie` [`comments`](https://news.ycombinator.com/item?id=47946899)
5. [**If I Could Make My Own GitHub**](https://matduggan.com/if-i-could-make-my-own-github/) `matduggan.com` [`comments`](https://news.ycombinator.com/item?id=47971771)
6. [**For Linux kernel vulnerabilities, there is no heads-up to distributions**](https://www.openwall.com/lists/oss-security/2026/04/30/10) `www.openwall.com` [`comments`](https://news.ycombinator.com/item?id=47965108)
7. [**Shai-Hulud Themed Malware Found in the PyTorch Lightning AI Training Library**](https://semgrep.dev/blog/2026/malicious-dependency-in-pytorch-lightning-used-for-ai-training/) `semgrep.dev` [`comments`](https://news.ycombinator.com/item?id=47964617)
8. [**Opus 4.7 knows the real Kelsey**](https://www.theargumentmag.com/p/i-can-never-talk-to-an-ai-anonymously) `www.theargumentmag.com` [`comments`](https://news.ycombinator.com/item?id=47951295)
9. [**Softmax, can you derive the Jacobian? And should you care?**](https://idlemachines.co.uk/essays/softmax) `idlemachines.co.uk` [`comments`](https://news.ycombinator.com/item?id=47927011)
10. [**CPanel and WHM Authentication Bypass – CVE-2026-41940**](https://labs.watchtowr.com/the-internet-is-falling-down-falling-down-falling-down-cpanel-whm-authentication-bypass-cve-2026-41940/) `labs.watchtowr.com` [`comments`](https://news.ycombinator.com/item?id=47969288)
11. [**Claude Code refuses requests or charges extra if your commits mention "OpenClaw"**](https://twitter.com/theo/status/2049645973350363168) `twitter.com` [`comments`](https://news.ycombinator.com/item?id=47963204)
12. [**How an oil refinery works**](https://www.construction-physics.com/p/how-an-oil-refinery-works) `www.construction-physics.com` [`comments`](https://news.ycombinator.com/item?id=47962548)
13. [**Can I disable all data collection from my vehicle?**](https://rivian.com/support/article/can-i-disable-all-data-collection-from-my-vehicle) `rivian.com` [`comments`](https://news.ycombinator.com/item?id=47967786)
14. [**I built a Game Boy emulator in F#**](https://nickkossolapov.github.io/fame-boy/building-a-game-boy-emulator-in-fsharp/) `nickkossolapov.github.io` [`comments`](https://news.ycombinator.com/item?id=47965503)
15. [**Maladaptive Frugality**](https://herbertlui.net/maladaptive-frugality/) `herbertlui.net` [`comments`](https://news.ycombinator.com/item?id=47942863)
16. [**Canonical/Ubuntu have been under DDoS for more than 15h**](https://status.canonical.com/#/incident/KNms6QK9ewuzz-7xUsPsNylV20jEt5kyKsd8A-3ptQEHpOd8VQ40ZQs-KD81fboQXeGZB94okNHdHBGlCv58Sw==) `status.canonical.com` [`comments`](https://news.ycombinator.com/item?id=47972213)
17. [**Roboticist-Turned-Teacher Built a Life-Size Replica of Eniac**](https://spectrum.ieee.org/roboticist-turned-teacher-eniac-replica) `spectrum.ieee.org` [`comments`](https://news.ycombinator.com/item?id=47948152)
18. [**We Run iSCSI over the Internet**](https://scsipub.com/blog/how-we-run-iscsi-over-the-internet) `scsipub.com` [`comments`](https://news.ycombinator.com/item?id=47948188)
19. [**Using a 1978 terminal in 2026 (DEC VT-100)**](https://nikhiljha.com/posts/vt100/) `nikhiljha.com` [`comments`](https://news.ycombinator.com/item?id=47939227)
20. [**You can beat the binary search**](https://lemire.me/blog/2026/04/27/you-can-beat-the-binary-search/) `lemire.me` [`comments`](https://news.ycombinator.com/item?id=47924912)
21. [**Reverse Engineering SimTower**](https://phulin.me/blog/simtower) `phulin.me` [`comments`](https://news.ycombinator.com/item?id=47937814)
22. [**New mechanical panoramic film camera from Jeff Bridges**](https://wideluxx.com) `wideluxx.com` [`comments`](https://news.ycombinator.com/item?id=47937033)
23. [**Belgium stops decommissioning nuclear power plants**](https://dpa-international.com/general-news/urn:newsml:dpa.com:20090101:260430-930-14717/) `dpa-international.com` [`comments`](https://news.ycombinator.com/item?id=47961319)
24. [**Full-Text Search with DuckDB**](https://peterdohertys.website/blog-posts/full-text-search-w-duckdb.html) `peterdohertys.website` [`comments`](https://news.ycombinator.com/item?id=47966254)
25. [**It’s Toasted**](https://yadin.com/notes/toasted/) `yadin.com` [`comments`](https://news.ycombinator.com/item?id=47971830)
